src/_playground/Routes.js

```diff
@@ -364,7 +364,7 @@ export class Routes extends Component {
                             <a
                                 aria-label='Home'
                                 className='frDocs-Menu__logo'
-                                href='/'>
+                                href='/home'>
```

Comment:
Not the end of the world, but this would make two places that would need to change if the default landing page ever changed.  This is currently handled by the `Redirect` component below.